homework/Truong_Duong/lesson_1/202_Happy_Number.py

```diff
@@ -0,0 +1,19 @@
+# Solution for https://leetcode.com/problems/happy-number/
+
+class Solution:
+    def isHappy(self, n: int) -> bool:
+        mset = {}
+
+        while n not in mset:
+            mset[n] = 1
+            currentSum = 0
+
+            while n != 0:
```

Comment:
consider move this logic to a separated function

Reply:
Thank you, I fixed it 🙇‍♂️